src/main/java/com/SAS/User/Coach.java

```diff
@@ -102,11 +108,15 @@ public void removeAssetFromTeam() {
 
     /**
      * The function adds personal page to the coach
-     *
      * @param description
+     * @return pageID - int
      */
-    public void addPage(String description) {
-        this.personalPage = new PersonalPage(description);
+    public int addPage(String description) {
+        if (description != null && !description.trim().isEmpty()) {
+            this.personalPage = new PersonalPage(description);
```

Comment:
line 115, those are 2 different erros, you should split it to 2 different cases in my opinion

Reply:
Why? it's the same case for me.

Reply:
because if the description is null, it didn't initiated.
if the description is empty string, it initiated but the vulue didn't saved.

Reply:
I understand, but in the two cases I don't want to add it to the personal page.

Reply:
In our opinion, both cases are invalid input so we wouldn't want to create a personal page which is blank. 
Therefore we would leave the implementation this way :)

Reply:
but the handale should be different, if there is a problem with null its different problem from empty string, you are wrong.